README.md

```diff
@@ -78,7 +78,8 @@ Clicking the button will now toggle it's aria-expanded state. CSS can be used to
 * `options.expandOnFocus`: whether the host should be focus activated (default: false)
 * `options.expandOnHover`: whether the host should be hover activated (default: false)
 * `options.focusManagement`: where focus should go (null, 'content', 'focusable', 'interactive', or ID reference) when click expander is activated with keyboard (default: null)
-* `options.hostSelector`: the query selector for the host element in relation to the widget (default: '.expander__host')
+* `options.hostSelector`: the query selector for the host element in relation to the widget (default: '.expander__host');
+* `options.hostContainerClass`: the class to which you want to add the `--expanded` modifier (default: '.expander__host-container'); useful for nested hosts; **_Example_**: setting this option to `tooltip` will make the expander add `.tooltip--expanded` to the first child element which has the `tooltip` class applied when the expander is expanded.
```

Comment:
How about instead of `options.hostContainerClass`, this was `options.expandedClass`, with a default of null. Then, when a value is set, that class name is added to the widget root tag whenever aria-expanded=true on the host.

For example:

```js
options = { expandedClass: 'tooltip--expanded' }
```

Would that work for the tooltip scenario?

Reply:
Oh, and the reason for having it null by default, is that originally I thought the expander added an `.expander` class to the root, but I see now that that class is just hardcoded in the docs html as a hook.

Reply:
What about the case where the class is used to find the "host"?

Reply:
Let's chat offline and go through this.